tests/impl.cpp

```diff
@@ -2026,7 +2026,16 @@ result_t test_mm_storel_pi(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_storeu_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    float *f = impl.mTestFloatPointer1;
+    float _a[4] = {1.0, 2.0, 3.0, 4.0};
+    __m128 a = _mm_load_ps(_a);
+
+    _mm_storeu_ps(f, a);
```

Comment:
You are storing the value of `a` to the memory address `f`.
`f` can be declared as an uninitialized array.
`a` should contain random value.

Rewrite as
```c
    float *_a = impl.mTestFloatPointer1;
    float f[4];
```